core/app/models/spree/store_credit.rb

```diff
@@ -28,6 +29,7 @@ class StoreCredit < Spree::Base
     delegate :email, to: :created_by, prefix: true
 
     scope :order_by_priority, -> { includes(:credit_type).order('spree_store_credit_types.priority ASC') }
+    scope :for_store, ->(store) { where(store_id: store.id) unless store.nil? }
```

Comment:
@szymoniwacz we don't need to define this scope - it's already automatically defined here: https://github.com/spree/spree/blob/master/core/app/models/spree/base.rb#L25

Reply:
You just need to add `has_many :store_credits` in `Store` model to make it work